sandbox/apparmor/apparmor.go

```diff
@@ -486,6 +486,10 @@ func AppArmorParser() (cmd *exec.Cmd, internal bool, err error) {
 	for _, dir := range filepath.SplitList(parserSearchPath) {
 		path := filepath.Join(dir, "apparmor_parser")
 		if _, err := os.Stat(path); err == nil {
+			abi30 := filepath.Join(dirs.GlobalRootDir, "/etc/apparmor.d/abi/3.0")
```

Comment:
this is such a nice find and thanks for getting this solved! -- however can we somehow move this ABI 3.0 path somewhere more explicit with an informative comment for both usages in this function?

Reply:
yes, having an unexported var for this would be good

Reply:
Done